src/main/java/nl/tudelft/ewi/devhub/server/web/resources/repository/AbstractProjectIssueResource.java

```diff
@@ -0,0 +1,200 @@
+import nl.tudelft.ewi.git.models.RepositoryModel;
+import nl.tudelft.ewi.git.web.api.RepositoriesApi;
+import nl.tudelft.ewi.git.web.api.RepositoryApi;
+
+/**
+ * 
+ * @author Aron Zwaan
+ *
+ */
+@Slf4j
+@Produces(MediaType.TEXT_HTML + Resource.UTF8_CHARSET)
+public abstract class AbstractProjectIssueResource extends AbstractIssueResource<Issue> {
+
+	protected Issues issues;
+	protected IssueBackend issueBackend;	
+	
+	
+	public AbstractProjectIssueResource( final TemplateEngine templateEngine, 
+			final User currentUser, 
+			final CommentBackend commentBackend,
+			final CommentMailer commentMailer, 
+			final RepositoriesApi repositoriesApi, 
+			final Issues issues, 
+			final IssueBackend issueBackend,
+			final Users users) {
+		
+		super(templateEngine, currentUser, commentBackend, commentMailer, repositoriesApi, users);
+		
+		this.issues = issues;
+		this.issueBackend = issueBackend;
+	}
+	
+	@GET
+	@Transactional
+	@Path("/issues")
+	public Response getIssues(@Context HttpServletRequest request) throws IOException{
+		RepositoryEntity repositoryEntity = getRepositoryEntity();
+		RepositoryApi repositoryApi = getRepositoryApi(repositoryEntity);
+		RepositoryModel repository = repositoryApi.getRepositoryModel();
+
+		List<Issue> openIssues = issues.findOpenIssues(repositoryEntity);
+		List<Issue> closedIssues = issues.findClosedIssues(repositoryEntity);
+
+		Map<String, Object> parameters = getBaseParameters();
+		
+		parameters.put("repository", repository);
+		parameters.put("openIssues", openIssues);
+		parameters.put("closedIssues", closedIssues);
+
+		List<Locale> locales = Collections.list(request.getLocales());
+		return display(templateEngine.process("courses/assignments/group-issues.ftl", locales, parameters));
+	
+	}
+	
+	@GET
+	@Transactional
+	@Path("/issues/create")
+	public Response openCreateIssuePage(@Context HttpServletRequest request) throws IOException{
+		
+		RepositoryEntity repositoryEntity = getRepositoryEntity();
+		RepositoryApi repositoryApi = getRepositoryApi(repositoryEntity);
+		RepositoryModel repository = repositoryApi.getRepositoryModel();
+		
+		Map<String, Object> parameters = getBaseParameters();		
+		parameters.put("repository", repository);
+		
+		List<Locale> locales = Collections.list(request.getLocales());
+		return display(templateEngine.process("courses/assignments/group-issue-create.ftl", locales, parameters));
+	}
+
+	@POST
+	@Transactional
+	@Path("/issues/create")
+	public Response createIssue(@Context HttpServletRequest request,
+			@FormParam("title") String title,
+			@FormParam("description") String description,
+			@FormParam("assignee") String assigneeNetID) throws IOException{
+		
+		Issue issue = new Issue();
+		
+		issue.setTitle(title);
+		issue.setDescription(description);
+		issue.setOpen(true);
+		
+		User assignee = users.findByNetId(assigneeNetID);
+		checkCollaborator(assignee);
+		
+		issue.setAssignee(assignee);
+		issue.setTimestamp(new Date());
```

Comment:
I believe this field has the `@Creation` annotation, so this assignment is not needed, and done by Hibernate under the hood.